hugo/content/showcase/domainmodel.html

```diff
@@ -1,11 +1,14 @@
 ---
```

Comment:
Why the rename? Any advantage or consequence?

Reply:
Changed from markdown to html so allow literal html content in this file. The old markdown file wasn't actually being used as well, as it was overridden by the prebuilt showcase contents on build.

This lets us add in the direct script tag that let us choose the language, syntax highlighting, example file contents, and anything else we need to demonstrate it. More granular control over the individual showcase, without having to define all the surrounding boilerplate and page structure.

Reply:
nice